src/register/components/PreactDecorator.tsx

```diff
@@ -155,6 +156,15 @@ function getBase(config: Config): string {
   return `<base href="${new URL(config.baseUrl).origin}">`;
 }
 
+function get3pIframeMeta(config: Config): string {
+  if (!config.baseUrl.startsWith("http://localhost")) {
```

Comment:
Should we do anything for PR deploy bot domains too?

Reply:
I don't know how those work, TBH, but I think something should be done there as well. @alanorozco can you explain how that works?

Reply:
I'll merge and we'll follow up on this subject separately.

Reply:
We should probably sync offline because I don't know how it works on `localhost` 🙂